### PR TITLE
refactoring text elements and renaming to code pal

### PR DESCRIPTION
**Pull request by lucasborin**

### CODE REVIEW

File: clean_code_main/clean_code_checks/y_check_empty_if_branches.clas.abap [hunk 1]
```diff
@@ -80,25 +80,13 @@ CLASS Y_CHECK_EMPTY_IF_BRANCHES IMPLEMENTATION.
...
 
-    y_message_registration=>add_message(
-      EXPORTING
-        check_name     = me->myname
-        text           = '[Clean Code]: Empty IF-Branch should be removed!'(102)
-        pseudo_comment = settings-pseudo_comment
-      CHANGING
-        messages       = me->scimessages ).
-  ENDMETHOD.                    "CONSTRUCTOR
+    set_check_message( '[Clean Code]: Empty IF-Branch should be removed!'(102) ).
```
Comment (lucasborin) [inline clean_code_main/clean_code_checks/y_check_empty_if_branches.clas.abap:88]: remove the reference to text elements

File: clean_code_main/clean_code_foundation/y_check_base.clas.xml [hunk 2]
```diff
@@ -13,6 +13,18 @@
...
+     <KEY>001</KEY>
+     <ENTRY>Check Description</ENTRY>
+     <LENGTH>30</LENGTH>
+    </item>
+    <item>
+     <ID>I</ID>
+     <KEY>002</KEY>
+     <ENTRY>Check Message</ENTRY>
+     <LENGTH>120</LENGTH>
+    </item>
```
Comment (lucasborin) [inline clean_code_main/clean_code_foundation/y_check_base.clas.xml:27]: no longer needed. Remove it.

File: clean_code_main/clean_code_checks/y_check_deprecated_key_words.clas.abap [hunk 3]
```diff
@@ -23,25 +23,13 @@ CLASS Y_CHECK_DEPRECATED_KEY_WORDS IMPLEMENTATION.
...
 
-    y_message_registration=>add_message(
-      EXPORTING
-        check_name     = me->myname
-        text           = '[Clean Code]: "&1" is deprecated'(102)
-        pseudo_comment = settings-pseudo_comment
-      CHANGING
-        messages       = me->scimessages ).
-  ENDMETHOD.                    "CONSTRUCTOR
+    set_check_message( '[Clean Code]: "&1" is deprecated'(102) ).
```
Comment (lucasborin) [inline clean_code_main/clean_code_checks/y_check_deprecated_key_words.clas.abap:31]: remove text element reference

**Review by lucasborin — COMMENTED** [review]
almost ok

**Review by eugen-guenther-sap — APPROVED** [review]

**Pull request merged**